lib/features/welcome/views/widgets/welcome_button.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+import 'package:kahootify/color_consts.dart';
+
+class WelcomeButton extends StatelessWidget {
```

Comment:
it's name doesn't feel appropriate. Refactor it to smth like ChooseModeButton or smth just not WelcomeButton please!!

Reply:
Ready